Tilde is treated as binary operator

Following code cause W503 to be reported, because ~ is treated as a binary operator, where in reality it is unary.
```
var = (1 &
       ~2)
```

Similarly, unary `-` is falsely being reported as a W503 violation.

```
[~/scratch]$ cat false_pep8.py
y = 75.0
z = 100.0
x = (y /
     -z)
[~/scratch]$ pep8 false_pep8.py
false_pep8.py:4:6: W503 line break before binary operator
[~/scratch]$ pep8 --version
1.7.0
```

--- a/pep8.py
+++ b/pep8.py
@@ -1017,13 +1017,14 @@ def break_around_binary_operator(logical_line, tokens):
     Okay: x = '''\n''' + ''
     Okay: foo(x,\n    -y)
     Okay: foo(x,  # comment\n    -y)
+    Okay: var = (1 &\n       ~2)
     """
     def is_binary_operator(token_type, text):
         # The % character is strictly speaking a binary operator, but the
         # common usage seems to be to put it next to the format parameters,
         # after a line break.
         return ((token_type == tokenize.OP or text in ['and', 'or']) and
-                text not in "()[]{},:.;@=%")
+                text not in "()[]{},:.;@=%~")
 
     line_break = False
     unary_context = True

